index.html

```diff
@@ -41,6 +45,177 @@ <h1 class="headline-heading">I'm Usman <br> Glad to see you!</h1>
             </ul>
         </section>
     </section>
+    <!-- Headline Section Ends Here -->
+
+    <!-- Works Section Starts Here -->
+    <!-- Card 1 starts -->
+
+    <section class="works-card">
+        <img src="./assets/works-1.png" alt="Recent Work">
+        <h1 class="works-card-heading-large">Tonic</h1>
+        <section class="works-card-client">
+            <p class="paragraph">CANOPY</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-role paragraph">Back End Dev</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-year paragraph">2015</p>
+        </section>
+        <p class="paragraph works-card-paragraph">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <section class="works-card-tags">
+            <p class="works-card-tag">html</p>
+            <p class="works-card-tag">css</p>
+            <p class="works-card-tag">javascript</p>
+        </section>
+        <button>See Project</button>
+    </section>
+
+    <!-- Card 1 Ends -->
+    <!-- Card 2 starts -->
+
+    <section class="works-card">
+        <img src="./assets/works-2.png" alt="Recent Work">
+        <h1 class="works-card-heading-large">Multi-Post Stories</h1>
+        <section class="works-card-client">
+            <p class="paragraph">CANOPY</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-role paragraph">Back End Dev</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-year paragraph">2015</p>
+        </section>
+        <p class="paragraph works-card-paragraph">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <section class="works-card-tags">
+            <p class="works-card-tag">html</p>
+            <p class="works-card-tag">css</p>
+            <p class="works-card-tag">javascript</p>
+        </section>
+        <button>See Project</button>
+    </section>
```

Comment:
I love these cards, and you can improve them by following the exact Figma template. Here are a few small points to consider.

1. On the client and position section you see the dots are a bit larger than the original, please make sure to reduce their size
2. Don't you think the height of the button is smaller than the original? if you increase it I'm sure it'll look better.

![image](https://user-images.githubusercontent.com/34779928/212384199-b81dec86-08fd-41e1-a934-85142cc29194.png)